core/src/main/java/com/google/errorprone/bugpatterns/AbortInOvercatch.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2013 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.errorprone.bugpatterns;
+
+import static com.google.errorprone.BugPattern.Category.JDK;
+import static com.google.errorprone.BugPattern.MaturityLevel.MATURE;
+import static com.google.errorprone.BugPattern.SeverityLevel.ERROR;
+import static com.google.errorprone.BugPattern.SeverityLevel.WARNING;
+
+import com.google.errorprone.BugPattern;
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.bugpatterns.BugChecker.TryTreeMatcher;
+import com.google.errorprone.fixes.SuggestedFix;
+import com.google.errorprone.matchers.Description;
+import com.google.errorprone.util.ASTHelpers;
+import com.sun.source.tree.LineMap;
+
+import static com.sun.source.tree.Tree.Kind.EXPRESSION_STATEMENT;
+
+import com.sun.tools.javac.tree.JCTree;
+import com.sun.tools.javac.tree.TreeInfo;
+import com.sun.source.tree.CatchTree;
+import com.sun.source.tree.StatementTree;
+import com.sun.source.tree.TryTree;
+import com.sun.source.tree.ExpressionTree;
+import com.sun.source.tree.ExpressionStatementTree;
+
+import com.sun.tools.javac.code.Symbol;
+import com.sun.tools.javac.code.Symbol.MethodSymbol;
+
+import java.util.List;
+
+/**
+ * A bug checker for abort in exception overcatch. See the following example: 
+ *
+ * <pre>
+ * try {
+ *   // stmt 1 throws exception B
+ *   // stmt 2 throws exception A
+ * } catch (Throwable t) {
+ *   // more than one unique exceptions fall into this catch -- overcatch
+ *   abort(); 
+ * }
+ * </pre>
+ *
+ * While it is a best practice to catch the precise exceptions, abort or System.exit()
```

Comment:
nit: in Javadoc, each paragraph after the first should begin with <p>.  Please also fix this for the other bugcheckers in this PR.